ci_tutorial/packages_pipeline/single_configuration.rst

```diff
@@ -0,0 +1,102 @@
+Package pipeline: single configuration
+======================================
+
+We will start with the most simple case, in which we only had to build 1 configuration, and that configuration 
+can be built in the current CI machine.
+
+As we described before while presenting the different server binary repositories, the idea is that package builds
+will use by default the ``develop`` repo only, which is considered the stable one for developer and CI jobs.
+
+Let's make sure we start from a clean state:
+
+.. code-block:: bash
+
+    $ conan remove "*" -c  # Make sure no packages from last run
+    $ conan remote remove "*"  # Make sure no other remotes defined
+    # Add only the develop repo, you might need to adjust this for your URL
+    $ conan remote add develop http://localhost:8081/artifactory/api/conan/develop
+
+
+The removal and addition of repos accross this tutorial can be a bit tedious, but it is important for the correct
+behavior. Also, there might be other configurations that can be even more efficient for some cases, like re-triggering
+a broken job because of CI malfunction, but we will keep it simple at the moment and try to focus on the main concepts.
+
+With this configuration the CI job could just do:
+
+.. code-block:: bash
+
+    $ cd ai
+    $ conan create . --build="missing:ai/*"
```

Comment:
maybe this saves one step?
```suggestion
    $ conan create ai --build="missing:ai/*"
```

Reply:
It has a purpose. The generated files in the commands are created inside the ``ai`` folder, if we move location. So they pollute less the overall project and it makes more evident that they are due to some source code changes of the ``ai`` package, but not others.